src/panels/config/storage/dialog-mount-view.ts

```diff
@@ -149,15 +153,36 @@ class ViewMountDialog extends LitElement {
         open
         scrimClickAction
         escapeKeyAction
-        .heading=${this._existing
-          ? this.hass.localize(
-              "ui.panel.config.storage.network_mounts.update_title"
-            )
-          : this.hass.localize(
-              "ui.panel.config.storage.network_mounts.add_title"
-            )}
+        .heading=${true}
```

Comment:
Add the text we use in the dialog header below here, because this is also used for a11y.